Makefile

```diff
@@ -178,24 +178,10 @@ ifneq ($(origin DOCKER_CMD),undefined)
 #       if there's a docker container, jump into it and run from there
 	$(DOCKER_CMD) make jstest
 else
-#        the 'dropdb' command may fail, hence the prefix minus-sign
-	-dropdb lsmb_test_api
-	perl -Ilib bin/ledgersmb-admin create \
-            $${PGUSER:-postgres}@$${PGHOST:-localhost}/$${PGDATABASE:-lsmb_test_api}#xyz
-	perl -Ilib -Iold/lib bin/ledgersmb-admin user create \
-            $${PGUSER:-postgres}@$${PGHOST:-localhost}/$${PGDATABASE:-lsmb_test_api}#xyz \
-				--username="$${UIUSER:-Jest}" \
-				--password="$${UIPASSWORD:-Tester}" \
-				--employeenumber=1 \
-				--country='Canada' \
-				--first_name="$${UIUSER:-Jest}" \
-				--last_name="$${UIPASSWORD:-Tester}" \
-				--permission='Full Permissions'
 	# Make sure API definition is current
 	./utils/devel/rebuild_api.sh
```

Comment:
Shouldn't this be a separate build target (with its own dependencies) on which the `jstest` target gets to depend?

Reply:
Fixed